### PR TITLE
[Chore](runtime filter) remove runtime filter ready_for_publish/publish_finally

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

remove runtime filter ready_for_publish/publish_finally

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exprs/runtime_filter_slots.h [hunk 1]
```diff
@@ -196,29 +197,33 @@ class RuntimeFilterSlotsBase {
         }
     }
 
-    // should call this method after insert
-    void ready_for_publish() {
+    bool read_finish_publish() {
```
Comment (HappenLee) [inline be/src/exprs/runtime_filter_slots.h:200]: ready

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exprs/runtime_filter_slots.h (hunk 1 above)
Comment (BiteTheDDDDt) [reply be/src/exprs/runtime_filter_slots.h]: fixed

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**